src/App.js

```diff
@@ -0,0 +1,20 @@
+// import logo from './logo.svg';
+// import './App.css';
```

Comment:
- [ ] Kindly remove this commented out code in order to make your codebase clean.